src/components/MapView/index.tsx

```diff
@@ -410,7 +410,10 @@ function MapView({ classes }: MapViewProps) {
       )}
       <MapboxMap
         // eslint-disable-next-line react/style-prop-object
-        style="mapbox://styles/eric-ovio/ckaoo00yp0woy1ipevzqnvwzi"
+        style={
+          process.env.REACT_APP_DEFAULT_STYLE ||
+          'mapbox://styles/eric-ovio/ckaoo00yp0woy1ipevzqnvwzi'
```

Comment:
does this style work with maplibre?

Reply:
If we are switching completely to maplibre, it might be worth creating/finding a similar style to `mapbox://styles/eric-ovio/ckaoo00yp0woy1ipevzqnvwzi` that is not hosted on mapbox

Reply:
I updated this to use the temporary Maptiler style on my account.